base_tier_validation/models/tier_validation.py

```diff
@@ -594,3 +624,22 @@ def get_view(self, view_id=None, view_type="form", **options):
             res["arch"] = etree.tostring(doc)
             res["models"] = frozendict(all_models)
         return res
+
+    def _notify_review_available(self, tier_reviews):
+        """method to notify when reaching pending"""
+        subscribe = "message_subscribe"
+        post = "message_post"
+        if hasattr(self, post) and hasattr(self, subscribe):
+            for rec in self.sudo():
+                users_to_notify = tier_reviews.filtered(
+                    lambda r, x=rec: r.definition_id.notify_on_pending
+                    and r.res_id == x.id
+                ).mapped("reviewer_ids")
+                # Subscribe reviewers and notify
+                getattr(rec, subscribe)(
+                    partner_ids=users_to_notify.mapped("partner_id").ids
+                )
+                getattr(rec, post)(
+                    subtype_xmlid=self._get_requested_notification_subtype(),
+                    body=rec._notify_requested_review_body(),
+                )
```

Comment:
If at this point in the code, it is certain that this method exists because it is not called directly?, it enhances readability.
```suggestion
                rec.message_subscribe(
                    partner_ids=users_to_notify.mapped("partner_id").ids
                )
                rec.message_post(
                    subtype_xmlid=self._get_requested_notification_subtype(),
                    body=rec._notify_requested_review_body(),
                )
```